services/telemetry/telemetry.go

```diff
@@ -902,6 +903,13 @@ func (ts *TelemetryService) trackPlugins() {
 		"plugins_with_settings":         settingsCount,
 		"plugins_with_broken_manifests": brokenManifestCount,
 	})
+
+	ts.srv.Go(func() {
+		pluginsEnvironment.RunMultiPluginHook(func(hooks plugin.Hooks) bool {
```

Comment:
As of today, there seems not to be any use for the return value, neither here nor in the hook. Can we make both functions that return no value?

Reply:
I'm not sure which function are you referring. Do you mean the return value of the functional argument of `RunMultiPluginHook`? If so, in e.g. https://github.com/mattermost/mattermost-server/blob/ee92bda912b78318e120e7efa46e72b71b991877/app/login.go#L165 it's used.

Reply:
Oh, I see. Then dismiss my comment.